pkg/sqlx/querybuilder_edit.go

```diff
@@ -410,7 +421,18 @@ func (qb *editQueryBuilder) FindCompletedEdits(votingPeriod int, minimumVotingPe
 			OR
 			(updated_at <= (now()::timestamp - (INTERVAL '1 second' * $1)) AND updated_at IS NOT NULL)
 			OR (
-				VOTES >= $2
+				$2 <= (
+					SELECT SUM(
+					  CASE
+						WHEN vote = 'ACCEPT' THEN 1
+						WHEN vote = 'REJECT' THEN -1
+						ELSE 0
+					  END
+					) as votesum
+					FROM edit_votes
+					WHERE edit_id = edits.id
+					AND (edits.updated_at IS NULL OR edit_votes.created_at > edits.updated_at)
+				)
```

Comment:
This appears to work as intended. The code is from the trigger function that was deleted.